example/pages/index.js

```diff
@@ -1,44 +0,0 @@
-import React from 'react'
```

Comment:
Deleting this index file which is the location for `appPageURL` since we are managing a global redirect.